<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAllOnlineUsers</class>
 <widget class="QDialog" name="GetAllOnlineUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>221</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Online Users</string>
   </property>
   <widget class="QListWidget" name="listOnlineUsersComm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>351</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>User Details</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection ID</string>
    </property>
    <widget class="QLabel" name="lblOnlineUsersContnIdComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>User Type</string>
    </property>
    <widget class="QLabel" name="lblOnlineUsersUsrTypeComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Name</string>
    </property>
    <widget class="QLabel" name="lblOnlineUsersNameComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Phone Number</string>
    </property>
    <widget class="QLabel" name="lblOnlineUsersPhNoComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>379</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Address</string>
    </property>
    <widget class="QLabel" name="lblOnlineUsersAddressComm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>150</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>550</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
